backend/applications/models.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Application(models.Model):
+    TYPE_CHOICES = (
+        (1, 'VOLUNTEER'),
+        (2, 'SPONSOR'),
+        (3, 'ENROLL')
```

Comment:
We might need to change this down the road, depending on what Kim wants to call this, but it should be fine for now.